src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs

```diff
@@ -134,11 +134,19 @@ private SyntaxNode Mutate(SyntaxNode currentNode, MutationContext context)
 
         private SyntaxNode MutateStaticConstructor(ConstructorDeclarationSyntax constructorDeclaration, MutationContext context)
         {
-            var trackedConstructor = constructorDeclaration.TrackNodes(constructorDeclaration.Body);
-            var mutatedBlock = (BlockSyntax) Mutate(constructorDeclaration.Body, context);
-            var markedBlock = MutantPlacer.PlaceStaticContextMarker(mutatedBlock);
+            var trackedConstructor = constructorDeclaration.TrackNodes((SyntaxNode) constructorDeclaration.Body ?? constructorDeclaration.ExpressionBody);
+            if (constructorDeclaration.ExpressionBody != null)
+            {
+                var markedBlock = Mutate(constructorDeclaration.ExpressionBody, context);
```

Comment:
The method `MutantPlacer.PlaceStaticContextMarker` should be called here. We mark static code so the coverage analyse can handle the static method.

Reply:
thanks @richardwerkman, I was going to raise this concern.
I have been thinking about this issue, and the only I could think of was to convert the _ExpressionBody_ to a (_BlockSyntax_) _Body_ so one can add the static marker.

Reply:
I was wondering about that.  Updated.